### PR TITLE
Do not set github token via command line

### PR DESCRIPTION
**Pull request by nicolasbock**
For now the user has to export their GitHub access token in the
environment. We can consider later to read a configuration file as an
alternative means to provide this token.

Closes-Bug: #5

### CODE REVIEW

File: send-pr [hunk 1]
```diff
@@ -43,21 +43,18 @@ def parse_options
...
-    end
   end.parse!
 
   raise OptionParser::ParseError, "missing --user" if options[:user].nil?
   raise OptionParser::ParseError, "missing --repository" if options[:repository].nil?
   raise OptionParser::ParseError, "missing --number" if options[:number].nil?
   raise OptionParser::ParseError, "missing --to" if options[:to].nil?
 
-  options[:access_token] = ENV['GITHUB_ACCESS_TOKEN'] if options[:access_token].nil?
+  options[:access_token] = ENV['GITHUB_TOKEN']
```
Comment (aspiers) [inline send-pr:53]: Why the change to `GITHUB_TOKEN`?

Comment (nicolasbock) [reply send-pr:53]: Maybe I followed the lead of hub a little bit too closely :) Either version is fine with me.

Comment (aspiers) [reply send-pr:53]: Oh, `hub` uses it - cool, that makes sense then.

**Review by aspiers — APPROVED** [review]
